src/projects.rs

```diff
@@ -1,5 +1,6 @@
 use crate::config::Config;
 use crate::items::{FormatType, Item};
+use crate::request::items_for_project;
```

Comment:
Please dont import the items_for_project and instead use when calling

```rust
request::items_for_project
```

For consistency and because it will confuse me and I'll be trying to figure out where it is in this module 😆

Reply:
Will try to keep it in mind(at this point it's a reflex) =)
Sounds like a good approach. And I think there are some conventional rules about this in Rust.

Structs - import all the way
Fn - use in place

or something like that =)